llvm/include/llvm/IR/Intrinsics.td

```diff
@@ -272,28 +272,29 @@ def IIT_F32  : IIT_VT<f32,  7>;
 def IIT_F64  : IIT_VT<f64,  8>;
 def IIT_V2   : IIT_Vec<2,   9>;
 def IIT_V4   : IIT_Vec<4,  10>;
-def IIT_V8   : IIT_Vec<8,  11>;
-def IIT_V16  : IIT_Vec<16, 12>;
-def IIT_V32  : IIT_Vec<32, 13>;
-def IIT_PTR  : IIT_Base<   14>;
-def IIT_ARG  : IIT_Base<   15>;
-
-def IIT_V64 : IIT_Vec<64, 16>;
-def IIT_MMX : IIT_VT<x86mmx, 17>;
-def IIT_TOKEN : IIT_VT<token, 18>;
-def IIT_METADATA : IIT_VT<MetadataVT, 19>;
-def IIT_EMPTYSTRUCT : IIT_VT<OtherVT, 20>;
-def IIT_STRUCT2 : IIT_Base<21>;
-def IIT_STRUCT3 : IIT_Base<22>;
-def IIT_STRUCT4 : IIT_Base<23>;
-def IIT_STRUCT5 : IIT_Base<24>;
-def IIT_EXTEND_ARG : IIT_Base<25>;
-def IIT_TRUNC_ARG : IIT_Base<26>;
-def IIT_ANYPTR : IIT_Base<27>;
-def IIT_V1 : IIT_Vec<1, 28>;
-def IIT_VARARG : IIT_VT<isVoid, 29>;
-def IIT_HALF_VEC_ARG : IIT_Base<30>;
-def IIT_SAME_VEC_WIDTH_ARG : IIT_Base<31>;
+def IIT_V6   : IIT_Vec<6,  11>;
```

Comment:
You should avoid lower numbering unless it is used often. `<=16` is for short encoders.

Reply:
Thanks! Moved to the end.